niworkflows/utils/spaces.py

```diff
@@ -448,6 +471,19 @@ def spaces(self):
         """Get all specified spaces."""
         return self._spaces
 
+    @property
+    def cached(self):
+        """Get cached spaces. If cached has not been set, raise `AttributeError`"""
+        if self._cached is None:
+            raise AttributeError("Spaces have not been cached")
```

Comment:
This should be a `ValueError` or a `RuntimeError` (I prefer the former):
```suggestion
            raise ValueError("Spaces have not been cached")
```